app/src/main/java/org/literacyapp/authentication/TrainingHelper.java

```diff
@@ -314,15 +314,29 @@ private File createAvatarFileFromStudentImage(StudentImage studentImage, Student
 
     /**
      * Find similar students
-     * Case 1: Student was added during fallback but in the meantime the same person has an existing StudentImageCollectionEvent and a new Student entry
+     *
+     *
      */
     public synchronized void findAndMergeSimilarStudents(){
         Log.i(getClass().getName(), "findAndMergeSimilarStudents");
         PreProcessorFactory ppF = new PreProcessorFactory(context);
+        TensorFlow tensorFlow = getInitializedTensorFlow();
+        findSimilarStudentsUsingAvatarImages(ppF, tensorFlow);
```

Comment:
No, because our library uses the default value 224 for resizing the image to 224 x 224 if no sharedPreference with the key "key_faceSize" exists (this key doesn't exist in our app because we don't have the Settings from the library included)